Core/VariantManager.swift

```diff
@@ -76,8 +76,8 @@ public struct Variant {
         Variant(name: "ma", weight: doNotAllocate, isIncluded: When.isIOS15EnglishWithOnDeviceSpeech, features: [.voiceSearch]),
         Variant(name: "mb", weight: doNotAllocate, isIncluded: When.isIOS15EnglishWithOnDeviceSpeech, features: []),
 
-        Variant(name: "mc", weight: 1, isIncluded: When.always, features: [.parameterStripping]),
-        Variant(name: "md", weight: 1, isIncluded: When.always, features: []),
+        Variant(name: "mc", weight: doNotAllocate, isIncluded: When.always, features: [.parameterStripping]),
```

Comment:
Shouldn't we remove the `.parameterStripping` case completely?

Reply:
Fine by me. I was following suit with the voice experiment above it. But perhaps that experiment has just been paused?